tests/acceptance/pageObjects/FilesPageElement/sharingDialog.js

```diff
@@ -563,8 +563,8 @@ module.exports = {
      * @return {*}
      */
     openExpirationDatePicker: function () {
-      this.waitForElementVisible('@expirationDateField')
-        .waitForElementNotPresent('@elementInterceptingCollaboratorsExpirationInput')
+      this.useCss()
+        .waitForElementVisible('@expirationDateField', 1000)
```

Comment:
please don't use hardcoded timeouts if possible. Search for `globals`, we have some timeouts defined in the nightwatch config.